library/classes/Prescription.class.php

```diff
@@ -479,11 +490,31 @@ function get_note() {
     }
 
     function set_drug($drug) {
+		if($GLOBALS['weno_rx_enable']){
+				$drug = explode("-", $drug); //striping the price from the drug name. 
+		        $drug = $drug[0];
+		}
         $this->drug = $drug;
+		
+       if($GLOBALS['weno_rx_enable']){
+        $sql = "SELECT NDC FROM erx_drug_paid WHERE drug_label_name LIKE ? ";
+        $val = array('%'.$drug.'%');
+        $getNDC = sqlStatement($sql, $val);
+		$ndc = sqlFetchArray($getNDC);
```

Comment:
better to use following to replace above 2 lines:
``````
$ndc = sqlQuery($sql, $val);